soh/soh/Enhancements/sfx-editor/SfxEditor.cpp

```diff
@@ -0,0 +1,359 @@
+    {NA_SE_VO_RT_LIFT,             {"Ruto Lift",                           "NA_SE_VO_RT_LIFT",               SEQ_SFX}},
+    {NA_SE_VO_RT_THROW,            {"Ruto Thrown",                         "NA_SE_VO_RT_THROW",              SEQ_SFX}},
+    {NA_SE_EN_NUTS_UP,             {"Scrub Emerge",                        "NA_SE_EN_NUTS_UP",               SEQ_SFX}},
+    {NA_SE_EN_AWA_BOUND,           {"Shabom Bounce",                       "NA_SE_EN_AWA_BOUND",             SEQ_SFX}},
+    {NA_SE_EN_SHELL_MOUTH,         {"Shellblade",                          "NA_SE_EN_SHELL_MOUTH",           SEQ_SFX}},
+    {NA_SE_EN_STALTU_DAMAGE,       {"Skulltula Damage",                    "NA_SE_EN_STALTU_DAMAGE",         SEQ_SFX}},
+    {NA_SE_EN_NUTS_THROW,          {"Spit Nut",                            "NA_SE_EN_NUTS_THROW",            SEQ_SFX}},
+    {NA_SE_VO_TA_CRY_0,            {"Talon Hmm",                           "NA_SE_VO_TA_CRY_0",              SEQ_SFX}},
+    {NA_SE_VO_TA_SURPRISE,         {"Talon Surprised",                     "NA_SE_VO_TA_SURPRISE",           SEQ_SFX}},
+    {NA_SE_SY_LOCK_ON,             {"Target Enemy",                        "NA_SE_SY_LOCK_ON",               SEQ_SFX}},
+    {NA_SE_SY_LOCK_ON_HUMAN,       {"Target Neutral",                      "NA_SE_SY_LOCK_ON_HUMAN",         SEQ_SFX}},
+    {NA_SE_SY_FSEL_CURSOR,         {"File Select Cursor",                  "NA_SE_SY_FSEL_CURSOR",           SEQ_SFX}},
+    {NA_SE_SY_FSEL_DECIDE_L,       {"File Select Choose",                  "NA_SE_SY_FSEL_DECIDE_L",         SEQ_SFX}},
+    {NA_SE_SY_FSEL_CLOSE,          {"File Select Back",                    "NA_SE_SY_FSEL_CLOSE",            SEQ_SFX}},
+    {NA_SE_IT_BOMB_EXPLOSION,      {"Bomb Explosion",                      "NA_SE_IT_BOMB_EXPLOSION",        SEQ_SFX}},
+    {NA_SE_EV_CHICKEN_CRY_A,       {"Chicken Cry",                         "NA_SE_EV_CHICKEN_CRY_A",         SEQ_SFX}},
+};
+
+void Draw_SfxTab(const std::string& tabKey, const std::map<u16, std::tuple<std::string, std::string, SeqType>>& map, SeqType type) {
+    const std::string hiddenTabKey = "##" + tabKey;
+    const std::string resetAllButton = "Reset All" + hiddenTabKey;
+    const std::string randomizeAllButton = "Randomize All" + hiddenTabKey;
+    if (ImGui::Button(resetAllButton.c_str())) {
+        for (const auto& [defaultValue, seqData] : map) {
+            const auto& [name, storageKeySuffix, seqType] = seqData;
+            if (seqType == type) {
+                const std::string storageKey = "gSfxEditor_" + storageKeySuffix;
+                CVar_SetS32(storageKey.c_str(), defaultValue);
+            }
+        }
+        SohImGui::RequestCvarSaveOnNextTick();
+    }
+    ImGui::SameLine();
+    if (ImGui::Button(randomizeAllButton.c_str())) {
+        std::vector<u16> values;
+        for (const auto& [value, seqData] : map) {
+            const auto& [name, storageKeySuffix, seqType] = seqData;
```

Comment:
looks like only `seqType` is being used here, maybe just grabbing that makes more sense?